app/javascript/src/components/category.vue

```diff
@@ -110,6 +106,9 @@ export default {
     selectedProduct() {
       return this.category.products[this.selectedProductIndex];
     },
+    priceIntervals() {
+      return process.env.PRICE_INTERVAL_LIMITS.split(',')
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'process' is not defined  no-undef